### ISSUE TITLE
Consider using Nextcloud Calendar 

### ISSUE THREAD

**Issue by nitn3lav** [wechange-eg/cosinnus-devops#77]
Users could then add their WeChange Cloud account to their preferred calendar app and have not only read but also write access to calendars from that app.

This could be implemented as 2 Calendars for each project, public and group only, which are shared with all project members (similar to the current shared cloud folder).
Cosinnus could then use [CalDAV](https://caldav.readthedocs.io/en/latest/) to access the Nextcloud calendars and display them in the UI.

Alternatives:
- Directly add CalDAV to the existing calendar.
- Implement a custom [Nextcloud calendar provider](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/groupware/calendar_provider.html)

I would be happy to help with the implementation.

**Comment by fenglisch** [comment]
Thank you for opening this issue and offering your help! Our plan is actually to integrate the Nextcloud Calendar into groups and projects as a replacement/improvement of the current calendar app. However, these plans aren't concrete yet, we don't even have a precise concept so far. We will definitely keep your offer and idea in the back of our heads and come back to you as soon as there are news.

**Comment by thobach** [comment]
At Dresden Zero we have regular meetings for the whole team every two weeks, but also on a sub-group level e.g. weekly. Today we need to enter every meeting manually, which is tedious. We'd like to have the option to enter recurring meetings.

With the move to Nextcloud it seems we'd get this option out of the box: https://docs.nextcloud.com/server/latest/user_manual/en/groupware/calendar.html#add-recurring-options

**Comment by fenglisch** [comment]
Yes, that's why we want to integrate the Nextcloud Calendar. See also https://wechange.de/cms/ufaq/13-koennen-wir-weitere-apps-in-der-nextcloud-aktivieren/
